scriptmodules/supplementary/esthemes.sh

```diff
@@ -235,6 +235,9 @@ function gui_esthemes() {
         'arcadeforge pixel-heaven-text'
         'arcadeforge 240p_Bubblegum'
         'arcadeforge 240p-honey'
+        'AndreaMav' 'arcade-crt'
```

Comment:
You should not separate the strings with single quotes in the middle. Pay attention to how is done in the other entries. You quote the entiry string separated by space like this:

    'AndreaMav arcade-crt`

Otherwise the scriptmodule breaks. You are not testing these changes before submitting aren't you?

Reply:
Sorry, i confirm... i just tested the theme, not the script :)

Reply:
thank you very much!